.github/ci.sh

```diff
@@ -163,14 +163,15 @@ install_system_deps() {
   install_yasm &
   wait
   export PATH=$PWD/$BIN:$PATH
-  echo "$PWD/$BIN" >> "$GITHUB_PATH"
+  echo "$BIN" >> "$GITHUB_PATH"
   is_exe "$BIN" z3 && is_exe "$BIN" cvc4 && is_exe "$BIN" yices && is_exe "$BIN" yasm
 }
 
 test_dist() {
   find_java
   pushd intTests
   for t in test0001 test0019_jss_switch_statement test_crucible_jvm test_ecdsa test_examples test_issue108 test_tutorial1 test_tutorial2 test_tutorial_w4; do echo $t >> disabled_tests.txt; done
+  env
```

Comment:
If we ever put any sensitive information in the environment (like secrets of some sort), this will leak that.

Reply:
Good point! We may want to remove it at some point. However, for now, it helps in debugging the thing that fails most often.